shortcut_composer/api_krita/actions/color_sampler_actions.py

```diff
@@ -0,0 +1,75 @@
+# SPDX-FileCopyrightText: © 2022 Wojciech Trybus <wojtryb@gmail.com>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from PyQt5.QtWidgets import (
+    QWidget,
+    QSpinBox,
+)
+
+from ..enums import Tool
+from ..core_api import KritaInstance
+Krita = KritaInstance()
+
+
+class ColorSamplerOptionsFinder:
+    """
+    Helper class for finding components related to Color Sampler Tool.
+
+    Stores elements of krita needed to control Color Sampler Tool:
+    - Widget Color Sampler tool options
+    - Blend property of Color Sampler
+
+    As the widget do not exist during plugin intialization phase,
+    fetching the elements needs to happen at runtime.
+    """
+
+    def __init__(self) -> None:
+        self._initialized = False
+
+        self._color_sampler_options: QWidget
+        self._blend_spinbox: QSpinBox
+
+    def ensure_initialized(self) -> None:
+        """Fetch widget and blend spinbox if not done already."""
+        if not self._initialized:
+            last_tool = Krita.active_tool
+            Krita.active_tool = Tool.COLOR_SAMPLER
+            self._color_sampler_options = self._fetch_color_sampler_options()
+            self._blend_spinbox = self._fetch_blend_spinbox()
+            self._initialized = True
+            Krita.active_tool = last_tool
+
+    def set_blend(self, blend: int) -> None:
+        """Set Color Sampler Blend percentage."""
+        self.ensure_initialized()
+
+        last_tool = Krita.active_tool
+        Krita.active_tool = Tool.COLOR_SAMPLER
+
+        blend = max(0, min(blend, 100))
```

Comment:
Mój trick na minmax:
`sorted(0, blend, 100)[1]`

Co jest czytelniejsze jest kwestią gustu - ten tutaj też jest ok :)

Reply:
Teee to sprytneeee. :O Choć chodziło ci pewnie o `sorted((o, blend, 100))[1]` Zmienię to, bo zawsze jak patrze max(min())) to dostaje zeza xD Powinni dodać clamp() do built-in.

Reply:
Chyba też zmienię 0 na 1, może nie ma to dużego sensu, ale uchroni przed dziwnym błędem? Co myślisz?